src/custom/api/gnosisSafe/index.ts

```diff
@@ -1,6 +1,10 @@
 import SafeServiceClient, { SafeInfoResponse, SafeMultisigTransactionResponse } from '@gnosis.pm/safe-service-client'
 import { registerOnWindow } from 'utils/misc'
 import { SupportedChainId as ChainId } from 'constants/chains'
+import { JsonRpcFetchFunc, Web3Provider } from '@ethersproject/providers'
+import EthersAdapter from '@gnosis.pm/safe-ethers-lib'
+// eslint-disable-next-line no-restricted-imports
+import { ethers } from 'ethers'
```

Comment:
I missed this the first time.
If I remember, this rule was added so there's tree shaking on ethers lib.
How does the bundle size look like after this change?